build/launcher/ssh_config

```diff
@@ -1,3 +1,5 @@
+Include /etc/ssh/ssh_config.d/*.conf
```

Comment:
is this just to slurp up any default stuff in container too? or do we actually put something the config.d dir?

Reply:
yeah, containerlab creates those per-lab-host entries in the `ssh_config.d` with setting the node username (as known by clab) and ssh options (if the node requires them)